pkgs/os-specific/bsd/openbsd/pkgs/csu.nix

```diff
@@ -11,6 +12,13 @@
 mkDerivation {
   noLibc = true;
   path = "lib/csu";
+  patches = [
+    (fetchpatch {
```

Comment:
A name or a description for this patch would be nice.

Reply:
Done